core/settings.py

```diff
@@ -30,9 +30,9 @@
     SECRET_KEY = ''.join(random.choice( string.ascii_lowercase  ) for i in range( 32 ))
 
 # Render Deployment Code
-#DEBUG = False
+DEBUG = False
```

Comment:
Could you explain why you needed to change DEBUG to False?

Reply:
When DEBUG = True, Django displays detailed error pages when exceptions occur, which could potentially expose sensitive information about the webapplication, environment, or server configuration to an attacker, which is why I put DEBUG = False.

Reply:
While technically true, in a development environment which the github repository represents it should be set to True
only once the application is deployed should it be set to False

Reply:
Yes, because we have to improve the security of the website which is already deployed, i put it FALSE so that it would be easier for the people who will add my part of the code to the main website.